src/main.ml

```diff
@@ -24,10 +38,22 @@ let depexts nv opams =
                 (V.to_string (V.Full.variable full_var)))
             variables
         in
-        if has_npm then OpamSysPkg.Set.Op.(names ++ depexts) else depexts)
-      OpamSysPkg.Set.empty
+        match has_npm with
+        | false -> npm_pkgs_and_constraints
+        | true -> (
+            match filter with
+            | OpamTypes.FOp (_a, `Eq, FString npm_constraint) ->
+                (names, npm_constraint) :: npm_pkgs_and_constraints
+            | _ ->
+                Printf.printf
+                  "Warning: package %s includes an invalid npm-version \
```

Comment:
This message explains what's wrong but not how to fix it.

Reply:
Added some suggestion to fix the issue in https://github.com/jchavarri/opam-check-npm-deps/pull/2/commits/e8d5c111ee0fc1fabda178429ecadad110586c54.